packages/tools/client-debugger/client-debugger-view/src/components/RenderSumaryTree.tsx

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import { Tree, TreeItem, TreeItemLayout } from "@fluentui/react-components/unstable";
+
+/**
+ * Input to {@link RenderSumaryTree}
+ */
+type RenderSummaryTreeProps = React.PropsWithChildren<{
+	/**
+	 * Header label created by {@link RenderLabel}.
+	 */
+	header: React.ReactElement | string;
+
+	/**
+	 * List of child React Elements populated by recursion.
+	 */
+	children: React.ReactElement[];
+}>;
+
+/**
+ * Outlays the React element populated by components in {@link TreeDataView}.
+ */
+export function RenderSummaryTree(props: RenderSummaryTreeProps): React.ReactElement {
```

Comment:
What do you think about just calling this "Tree"? I would also recommend moving this and the RenderLabel component into the `utility-components` directory, they aren't specific to the data visualizer stuff in this directory.

Reply:
Re-naming it to `Tree` incurs collision with `fluentUI`'s `Tree`. Perhaps to `ContainerTree`?

Reply:
You can always do something like `import { Tree as FluentTree } from ...` to differentiate, if that helps. But either way works for me, so long as we document what it does 🙂

Reply:
Ah I learned something new. Changed to using `FluentUI`.